gnovm/cmd/gno/test.go

```diff
@@ -145,75 +150,72 @@ func execTest(cfg *testCfg, args []string, io *commands.IO) error {
 		}()
 	}
 
+	subPkgs, err := gnomod.SubPkgsFromPaths(paths)
+	if err != nil {
+		return fmt.Errorf("list sub packages: %w", err)
+	}
+
 	buildErrCount := 0
 	testErrCount := 0
-	for _, pkgPath := range pkgPaths {
+	for _, pkg := range subPkgs {
 		if cfg.precompile {
 			if verbose {
-				io.ErrPrintfln("=== PREC  %s", pkgPath)
+				io.ErrPrintfln("=== PREC  %s", pkg.Dir)
```

Comment:
my main point is that here, we should use pkg.Path, which is not the same as any Dir. because "PkgPath/pkgPath/pkgpath/pkg.Path" are all the same "package path" identifier, whereas anything that ends with "Dir" is a filesystem location. They can diverge. Also a package path starts with a domain unless it is special, like a stdlibs package or something special like "main". Dir is a different beast based on the filesystem. Either absolute or relative, not something that starts with "gno.land" nor "main". It might be "." or "./something" though, or "/something". We should ideally make sure all Dirs start with a dot or slash.

Reply:
I agree we should print pkg.Path instead of Dir. original var `pkgPath` also contains Dir. So this didn't changed anything. 

Will fix this. But in another. (Out of scope for this PR)